ledger/src/blockstore.rs

```diff
@@ -136,10 +136,27 @@ impl std::fmt::Display for InsertDataShredError {
     }
 }
 
+/// A "complete data set" is a range of [`Shred`]s that combined in sequence carry a singled
+/// serialized [`Vec<Entry>`].
+///
+/// Services such as `solana_core::window_service::WindowService` for a TVU, and
```

Comment:
nit: think the full path is unnecessary should we ever move these into separate crates. Just a reference to the service should be sufficient